supplementary_files/lambdas/write_results_report/lambda_function.py

```diff
@@ -58,13 +58,23 @@ def simple_pk_query(*,
         
         return items
 
-def determine_compliance(*,InfractionItems):
-    return not bool(InfractionItems)
+def determine_compliance(*,InfractionItems,AllNestedSfnsSucceeded):
```

Comment:
FYI PEP 8 defines conventions for Python code style and nearly all code checking tools, big open source projects, and generally Python projects will be expected to follow these standards.

In this case I see a violation of the following:

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.
> Variable names follow the same convention as function names.

The best way I've seen to follow/learn these rules is:

https://flake8.pycqa.org/en/latest/

Reply:
Addressed